pkg/ctl/install/flux.go

```diff
@@ -68,17 +68,17 @@ func installFluxCmd(cmd *cmdutils.Cmd) {
 	})
 
 	cmd.FlagSetGroup.InFlagSet("Flux installation", func(fs *pflag.FlagSet) {
-		fs.StringVar(&opts.GitURL, "git-url", "",
-			"URL of the Git repository to be used by Flux, e.g. git@github.com:<github_org>/flux-get-started")
-		fs.StringVar(&opts.GitBranch, "git-branch", "master",
+		fs.StringVar(&opts.GitOptions.URL, "git-url", "",
+			"SSH URL of the Git repository to be used by Flux, e.g.: git@github.com:<github_org>/<repo_name>")
```

Comment:
Nit: I haven't seen a colon (`:`) used after `e.g.` before, I think we can remove it
```suggestion
			"SSH URL of the Git repository to be used by Flux, e.g. git@github.com:<github_org>/<repo_name>")
```

Reply:
We have `e.g.:` in other flags' descriptions. I'll harmonise this.